etc/README.hbs

```diff
@@ -39,13 +39,13 @@ modify an existing installation) and select *Common Tools for Visual C++* during
 Only the following version combinations with the [MongoDB Node.js
 Driver](https://github.com/mongodb/node-mongodb-native) are considered stable.
 
-|               | `kerberos@1.x` | `kerberos@2.x` | `kerberos@7.x` |
+| | `kerberos@1.x` | `kerberos@2.x` | `kerberos@7.x` |
 | ------------- | -------------- | -------------- | -------------- |
-| `mongodb@7.x` | N/A            | N/A            | ✓              |
-| `mongodb@6.x` | N/A            | ✓              | N/A            |
-| `mongodb@5.x` | ✓              | ✓              | N/A            |
-| `mongodb@4.x` | ✓              | ✓              | N/A            |
-| `mongodb@3.x` | ✓              | N/A            | N/A            |
+| `mongodb@7.x` | N/A | N/A | ✓ |
```

Comment:
Do we want the tables auto-formatted like this to remove whitespace? I think I'd prefer to have better readability inside the raw markdown (that goes for both tables - this one and the platform list itself below)

Reply:
This was auto-formatted for me. I'll revert it for now.